e2e/login/login.navigation.service.ts

```diff
@@ -0,0 +1,43 @@
+import { browser } from 'protractor';
+import { LoginPage } from './login.po';
+
+declare const allure: any;
+
+export class LoginNavigationService {
+    public static login(loginPage: LoginPage, useWrongPassword: boolean = false, clickEnter: boolean = true) {
+        loginPage.getUsernameField()
+            .sendKeys(browser.params.login.user);
+        if (!useWrongPassword) {
+            loginPage.getPasswordField()
+                .sendKeys(browser.params.login.password);
+        } else {
+            loginPage.getPasswordField()
+                .sendKeys('PassProvidedByTesting');
+        }
+        if (clickEnter) {
+            loginPage.getEnterButton()
+                .click();
+        }
+    }
+
+    public static setUserNameAndPassword(loginPage: LoginPage, userName: string, password: string) {
```

Comment:
The purpose of the navigation service is just to have the navigation at one place. It would be better to remove the "allure.createStep" from this file.  According to that, the method setUserNameAndPassword must be in the login.e2e-spec.ts.

Reply:
done